### ISSUE TITLE
ckb-next : The "*" script could not be loaded. please check your animation directory

### ISSUE THREAD

**Issue by Alkeryn** [NixOS/nixpkgs#67371]
Animation don't work anymore, and i get this error message
![image](https://user-images.githubusercontent.com/11599075/63630008-cfd60280-c616-11e9-86e5-c088120d0854.png)

Also it needs to be patched so it works with Qt stdenv.mkDerivation -> mkDerivation
and add wrapQtAppsHook

Lastly it would be nice for the package to be updated to 0.4.0 from 0.3.2

**Comment by Alkeryn** [comment]
@kierdavis As you are the maintainer, any idea ?

**Comment by kierdavis** [comment]
Hmm, yeah there are definitely problems with ckb-next on master - I haven't noticed them since I run the release channel of nixpkgs on all my machines (and also because the machine with all my Corsair equipment has been unplugged for months...).

I'm seeing this when I try to start `ckb-next`:

```
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Aborted
```

Is this what your point about Qt is referring to, or something else?

There's already a PR for updating to 0.4.0 (#60262), it just needs some TLC from the nixpkgs maintainers.

**Comment by kierdavis** [comment]
Huh, okay that error only occurs specifically when I build from the `nixpkgs-unstable` channel. The `nixos-unstable` channel is fine, as is git master.

**Comment by Alkeryn** [comment]
> Hmm, yeah there are definitely problems with ckb-next on master - I haven't noticed them since I run the release channel of nixpkgs on all my machines (and also because the machine with all my Corsair equipment has been unplugged for months...).
> 
> I'm seeing this when I try to start `ckb-next`:
> 
> ```
> qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
> This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
> 
> Aborted
> ```
> 
> Is this what your point about Qt is referring to, or something else?
> 
> There's already a PR for updating to 0.4.0 (#60262), it just needs some TLC from the nixpkgs maintainers.

The issue is now closed, i fixed both with #67417 and #67495 ^
